### PR TITLE
Switch docs to mamba and Mambaforge

### PR DESCRIPTION
**Pull request by xylar**
Addresses #427

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -112,45 +118,57 @@ acme1), you will get an SSL error unless you disable the SSL verification:
...
 See :ref:`"Installation in a Conda Environment" <conda_environment>` section above for
 installing conda.
-Create a new Anaconda environment with ``zppy`` installed and activate it: ::
 
-   $ conda create -n zppy_env -c e3sm -c conda-forge zppy
-   $ source activate zppy_env
+Create a new conda environment with ``zppy`` installed and activate it: ::
 
-Or you can install ``zppy`` in an existing environment. ::
+   mamba create -n zppy_env zppy
```
Comment (forsyth2) [inline docs/source/getting_started.rst:140]: @xylar Thanks for putting this together. I'm following along with the instructions but at this command I get `bash: mamba: command not found`.

Comment (xylar) [reply docs/source/getting_started.rst:140]: Did you install Mambaforge?  If not, did you got to the `Troubleshooting` section below?

**Comment by xylar** [comment]
I've gone ahead and reorganized in a way that I hope is more intuitive for you.

**Review by forsyth2 — APPROVED** [review]
Thanks @xylar. Following these instructions, I was able to set up both the production environment and the development environment. I still need to make the changes mentioned in #397 however.

**Comment by xylar** [comment]
@forsyth2, I see, I had forgotten how the issues were interrelated.  You are right that the yaml file should also be updated to switch `sphinx-multiversion` to come from conda-forge, not pip.  That seems like a trivial change.

I think everything else in #397 relates to GitHub Actions (which is installing a lot of stuff from pip).  I didn't see anywhere else that pip was being used in zppy (besides `pip install .`, which is fine).  So I think other than maybe the one `sphinx-multiversion` change, #397 is not related to this change, which is about how developers interact with zppy, rather than what CI does.

**Comment by forsyth2** [comment]
@xylar Is this ready to merge?

**Comment by xylar** [comment]
@forsyth2, yes, as far as I'm concerned, this is ready to merge.

**Pull request merged**